crates/re_data_source/src/data_loader/loader_external.rs

```diff
@@ -37,10 +37,12 @@ pub static EXTERNAL_LOADER_PATHS: Lazy<Vec<std::path::PathBuf>> = Lazy::new(|| {
         .flat_map(|paths| paths.split(':').map(ToOwned::to_owned).collect::<Vec<_>>())
         .map(std::path::PathBuf::from);
 
-    let executables: ahash::HashSet<_> = dirpaths
-        .into_iter()
-        .flat_map(|dirpath| {
-            WalkDir::new(dirpath).into_iter().filter_map(|entry| {
+    let mut executables = HashMap::<String, Vec<std::path::PathBuf>>::default();
+    for dirpath in dirpaths {
+        let paths = WalkDir::new(dirpath)
+            .max_depth(1) // don't search recursively
```

Comment:
At this point you might as well use `std::fs::read_dir` (less dependencies)

Reply:
I removed the use of `walkdir` in this instance but we still use it for actually recursive walks in other places so cannot get rid of the dependency entirely